src/main/java/org/elasticsearch/action/count/CountRequestBuilder.java

```diff
@@ -152,4 +154,19 @@ private QuerySourceBuilder sourceBuilder() {
         }
         return sourceBuilder;
     }
+
+    @Override
+    public String toString() {
+        if (sourceBuilder != null) {
+            return sourceBuilder.toString();
+        }
+        if (request.source() != null) {
+            try {
+                return XContentHelper.convertToJson(request.source().toBytesArray(), false, true);
+            } catch(Exception e) {
+                return "{ \"error\" : \"" + ExceptionsHelper.detailedMessage(e) + "\"}";
+            }
+        }
+        return new QuerySourceBuilder().toString();
```

Comment:
This is printing an empty thing.

```java
        SearchRequestBuilder builder = client().prepareSearch("index").setTypes("type");
        System.out.println("builder = " + builder);
```

Gives 

```
builder = { }
```

Is that expected @javanna? Or should we wait for #9962 for a fix?